libs/designer/src/lib/core/actions/bjsworkflow/add.ts

```diff
@@ -203,7 +203,22 @@ const initializeOperationDetails = async (
   // Re-update settings after we have valid operation data
   const rootNodeId = getRootNodeDetails(state.workflow, connectorId);
   const operation = getState().workflow.operations[nodeId];
-  const settings = getOperationSettings(isTrigger, operationInfo, initData.nodeOutputs, manifest, swagger, operation, rootNodeId);
+
+  const connectionReferences = state.connections.connectionReferences;
+  if (state.connections.connectionReferences !== undefined) {
```

Comment:
No need for the `!== undefined` here, can just do `if (!state.connections.connectionReferences) {`.